### ISSUE TITLE
JoJo false positives

### ISSUE THREAD

**Issue by Uma410** [ThioJoe/YT-Spammer-Purge#750]
### Filter Mode

Auto-Smart Mode

### Select the Problem

False Positive

### Spammer Example / Sample

https://www.youtube.com/watch?v=pmj9R1qIokw&lc=UgxHt7ySYxWIgUW_bQl4AaABAg.9BG9Jq_8aFA9BGeqAZR1Bb

also, this is the rest of the log:

[Spam_Log_2022-03-20_00-58-45.zip](https://github.com/ThioJoe/YT-Spammer-Purge/files/8310370/Spam_Log_2022-03-20_00-58-45.zip)

### Video / Post Link

https://www.youtube.com/watch?v=pmj9R1qIokw

### (Optional) Additional Info / Context

It's just talking about JoJo with some _very_ atrocioius grammar

**Comment by Firecul** [comment]
Why do people zip files, we don't pay for Github's HDDs, it just makes the files look suspicious in my eyes.


---


[Spam_Log_2022-02-17_19-00-33.txt](https://github.com/ThioJoe/YT-Spammer-Purge/files/8310546/Spam_Log_2022-02-17_19-00-33.txt)
(Same file)

**Comment by Firecul** [comment]
Tbf it *does* mark his as a possible false positive
![WindowsTerminal_Windows_PowerShell (39)](https://user-images.githubusercontent.com/34950371/159146778-a148b996-2859-454b-9b85-80d43129f34f.png)

**Comment by ThioJoe** [comment]
Hm i'll have to look at why it flagged. Super long comments are simply more likely to have something that triggers it

**Comment by KendallDoesCoding** [comment]
> Why do people zip files, we don't pay for Github's HDDs, it just makes the files look suspicious in my eyes.

Yeah true, for all we know someone can program a virus that if you download the zip it adds a virus to your laptop/pc.
